### PR TITLE
Add tutorial on using BTP service operator

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Add a tutorial describing how to use BTP service operator in Kyma

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/11290

### CODE REVIEW

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+```
+kubectl create ns sap-btp-operator
+```
+
+2. For the BTP service operator to work, you must [disable Istio sidecar proxy injection](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/smsh-01-istio-disable-sidecar-injection#documentation-content) that is enabled on the Kyma clusters by default. To do so, run this command:
+```
+kubectl label namespace sap-btp-operator istio-injection=disabled
+```
+
+3. Install and set up the [SAP BTP service operator](https://github.com/SAP/sap-btp-service-operator) in your Kyma cluster. Use the option without cert-manager.
```
[the remark this answers is not in the document]
Comment (klaudiagrz) [reply docs/03-tutorials/00-service-management/smgt-01-deploy-service.md]: Applied and tested, it works well! 🎉

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+This tutorial describes how you can deploy a simple SAP BTP audit log service in your Kyma cluster using the SAP BTP service operator.
+
+## Prerequisites
+
+- [Kyma cluster](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/02-install-kyma/) running on Kubernetes version 1.19 or higher
+- SAP BTP [Global Account](https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/d61c2819034b48e68145c45c36acba6e.html?locale=en-US) and [Subaccount](https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/55d0b6d8b96846b8ae93b85194df0944.html?locale=en-US)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) v1.17 or higher
+- [helm](https://helm.sh/) v3.0 or higher
+- [jq](https://stedolan.github.io/jq/download/)
+<!-->Should we list it here?<--->
```
Comment (PK85) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:14]: yes

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+## Steps
+
+>**NOTE:** For the BTP service operator to work, you must [disable Istio sidecar proxy injection](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/smsh-01-istio-disable-sidecar-injection#documentation-content) which is enabled on the Kyma clusters by default. The commands in this tutorial already mark the created Namespaces with the **istio-injection=disabled** flag, so you don't have to do this manually.  
+
+1. Create a Namespace and install cert-manager on it. Cert-manager is an obligatory tool that will manage certificates in your cluster. Run:
```
Comment (PK85) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:20]: "Cert-manager is an obligatory tool that will manage certificates in your cluster" -> change to: "cert manager is required for btp-operator. Omit that step if you have cert-manager already installed"

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    ```
+    kubectl create ns cert-manager
+    kubectl label namespace cert-manager istio-injection=disabled
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+    echo "Wait two minutes. For details, read: https://cert-manager.io/docs/concepts/webhook/#webhook-connection-problems-shortly-after-cert-manager-installation"
+    sleep 120
+    ```
+
+2. Obtain the access credentials for the SAP BTP service operator as described in step 2 of [SAP BTP Operator setup](https://github.com/SAP/sap-btp-service-operator#setup) document. Then, save the obtained credentials to the `creds.json` file.
+<!-->Should we link to btp docs or just provide instructions here?<--->
```
Comment (PK85) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:31]: link, as u did

**Review by PK85 — DISMISSED** [review]

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+      name: btp-audit-log-instance
+      namespace: default
+    spec:
+      serviceOfferingName: auditlog-api
+      servicePlanName: default
+      externalName: btp-audit-log-instance
+    EOF
+    ```
+
+    >**TIP:** You can find values for the **serviceOfferingName** and **servicePlanName** parameters in Service Marketplace of SAP BTP Cockpit. Click on the service's tile and find **name** and **Plan** respectively. The value of the **externalName** parameter must be unique.
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:59]:
```suggestion
    >**TIP:** You can find values for the **serviceOfferingName** and **servicePlanName** parameters in the Service Marketplace of the SAP BTP Cockpit. Click on the service's tile and find **name** and **Plan** respectively. The value of the **externalName** parameter must be unique.
```

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+5. To see the output, run:
+
+    ```
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:63]:
```suggestion
    ```bash
```

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+    kubectl get serviceinstances.services.cloud.sap.com btp-audit-log-instance -o yaml
+    ```
+
+    You can see the status "created" and the message "ServiceInstance provisioned successfully".
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:67]:
```suggestion
    You can see the status `created` and the message `ServiceInstance provisioned successfully`.
```

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+    spec:
+      serviceInstanceName: btp-audit-log-instance
+      externalName: btp-audit-log-binding
+      secretName: btp-audit-log-binding
+    EOF
+    ```
+
+7. To see the output, run:
+
+    ```
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:87]:
```suggestion
    ```bash
```

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+    kubectl get servicebindings.services.cloud.sap.com btp-audit-log-binding -o yaml
+    ```
+
+    You can see the status "created" and the message "ServiceBinding provisioned successfully".
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:91]:
```suggestion
    You can see the status `created` and the message `ServiceBinding provisioned successfully`.
```

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+8. You can now use a given service in your Kyma cluster. To see credentials, run:
+
+    ```
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:95]:
```suggestion
    ```bash
```

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+    kubectl get secret btp-audit-log-binding -o yaml
+    ```
+
+9. Clean up your resources:
+
+    ```
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:101]:
```suggestion
    ```bash
```

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+    kubectl delete servicebindings.services.cloud.sap.com btp-audit-log-binding
+    kubectl delete serviceinstances.services.cloud.sap.com btp-audit-log-instance
+    helm delete btp-operator -n sap-btp-operator
+    kubectl delete -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+    kubectl delete ns cert-manager
+    kubectl delete ns sap-btp-operator
+    ```
+
+>**TIP:** You can use Kyma Dashboard to create and manage resources such as Service Instances and Service Bindings. To do so, navigate to your Namespace view and go to the **Service Management** tab in the left navigation. Still, you need to obtain service details, such as service name and plan, from BTP Cockpit.
```
Comment (mmitoraj) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:110]:
```suggestion
>**TIP:** You can use Kyma Dashboard to create and manage resources such as Service Instances and Service Bindings. To do so, navigate to your Namespace view and go to the **Service Management** tab in the left navigation. Still, you need to obtain service details, such as service name and plan, from the BTP Cockpit.
```

**Review by mmitoraj — COMMENTED** [review]
I added my comments :)

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
+---
+title: Deploy an SAP BTP service in your Kyma cluster
+---
+
+This tutorial describes how you can deploy a simple SAP BTP audit log service in your Kyma cluster using the SAP BTP service operator.
```
[the remark this answers is not in the document]
Comment (PK85) [reply docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:5]: @mmitoraj it's not a case anymore, we can mention SAP and SAP products

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+    kubectl create ns cert-manager
+    kubectl label namespace cert-manager istio-injection=disabled
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+    echo "Wait two minutes. For details, read: https://cert-manager.io/docs/concepts/webhook/#webhook-connection-problems-shortly-after-cert-manager-installation"
+    sleep 120
+    ```
+
+2. Obtain the access credentials for the SAP BTP service operator as described in step 2 of [SAP BTP operator setup](https://github.com/SAP/sap-btp-service-operator#setup) document. Then, save the obtained credentials to the `creds.json` file.
+
+3. Create a Namespace and install [SAP BTP service operator](https://github.com/SAP/sap-btp-service-operator) on it:
```
[the remark this answers is not in the document]
Comment (klaudiagrz) [reply docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:32]: Good point, thanks for catching that! This point was originally the first and the link stayed in this sentence. But I agree it should be moved above 🙂

File: docs/03-tutorials/00-service-management/smgt-01-deploy-service.md (hunk 13 above)
Comment (klaudiagrz) [inline docs/03-tutorials/00-service-management/smgt-01-deploy-service.md:5]:
```suggestion
This tutorial describes how you can deploy a simple SAP BTP audit log service in your Kyma cluster using the [SAP BTP service operator](https://github.com/SAP/sap-btp-service-operator).
```

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**